README.md

```diff
@@ -170,3 +170,7 @@ Commonly encountered issues:
 * Not connecting when OMF support is enabled.
     * We have found that you may need to rerun the PI Web API Admin Utility to ensure that PI data archive is actually connected.
     * This is especially true if the connector was working properly and then stopped working after restarting the OSIsoft server machine.
+
+## Code Formatting
+
+Source code has been formatted using the following online tool: https://www.techiedelight.com/tools/java
```

Comment:
I am going to strongly protest the use of an online format and not a vs code or eclipse based solution.

Reply:
Its probably best to drop the auto format from this pull request if the goal is to have it done by tomorrow. I can probably take on the task of setting up good settings for eclipse.